trade-finance-manager-ui/templates/_macros/pagination.njk

```diff
@@ -5,11 +5,11 @@
   {% set paginationRoute = params.paginationRoute %}
   {% set queryString = params.queryString %}
 
-  <div class="pagination" role="navigation">
+  <div class="pagination" data-cy="pagination">
 
     {% if totalPages > 1 %}
 
-      <div data-cy="pagination" role="navigation">
+      <nav>
```

Comment:
should we give the nav a data-cy?

Reply:
I don't think so, this was actually semantically incorrect - the parent div (which now has `data-cy`) is the actual wrapping pagination div.